Neil weather start

neil-mckenna opened the pull request:
This is my starting code I have been working on.

It has a WeatherPage which can be navigated from the main page
, MVVM Code structure and namespacing,

It also has utility converter classes to handle different data types for xaml 

I have relevant Weather App code and 

I have SQL code to insert 

This display weather data from locations and at different times and you can cycle through all the locations to view the data

Please review this code and add this to main so I can move on to other sections

neil-mckenna committed:
neils-weather-start

My Initial Code entry, Weather App Page display, Database query setup, MVVM Setup and namespacing and utility converters, README Updates, Changes to Navigation,

neil-mckenna committed:
Added the Class UML

Class UML for the weather section

neil-mckenna committed:
Data, and Scroll

-Added more Database entries with smart  table create and destruction,

-Added Buttons to navigate the locations,

-Added Sql methods to dynaicmally  change the size of the entries for going up and down the iterations

-Changed LocationIds to a property

- Added more styling to teh WeatherPage.xaml

-- Added Microsoft.data.Sqlclient for a more up to date library

MiTsErRobot1943 reviewed (changes requested):
Could use some better comments to make the code easier to understand. Also to follow the asssement guideance I think your to add in html documentation

neil-mckenna committed:
Summaries, DocFX for HTML page

Added DocFX to generate html content for code for my classes and models, converters and viewmodels,

Update the code with better summary comments,

In the _site/index.html we have the code display with correct data class models,

there a lot of files was created when docfx builds from xml

MiTsErRobot1943 reviewed (approved):
Good use of coding principles and design patterns
